wgpu-core/src/command/render.rs

```diff
@@ -321,25 +333,32 @@ impl<G: GlobalIdentityHandlerFactory> Global<G> {
 
         let color_attachments = targets.colors
             .iter()
-            .take_while(|at| at.attachment != id::TextureViewId::ERROR)
+            .take_while(|at| at.attachment != 0)
```

Comment:
Although it probably won't happen often, it might still be nice to have associated constants for `const ERROR: u64 = 0u64` on `Id` instead of comparing against `0`

Reply:
But the whole point here is that you can't have `ERROR` associated constant like this, since `Id` can't be that value ever :)

Reply:
Oh ok sounds good, I thought we were accepting an externally provided `u64` here and trying to check if it's `None`

Reply:
oh wait, you wrote `ERROR: u64`. Yes, sure, we could have such a constant, maybe under a different name.

Reply:
Yeah, maybe `NONE: u64` or something